graylog2-server/src/test/java/org/graylog2/indexer/indices/IndicesIT.java

```diff
@@ -132,6 +148,18 @@ public void testClose() {
         //assertThat(getIndexState(index)).isEqualTo("close");
     }
 
+    @Test
+    public void testClosedIndices() {
```

Comment:
I would suggest to rename this to `findsClosedIndices` and also create a third index and not close it to verify that *only* closed indices are found.

Reply:
:heavy_check_mark: